src/content/docs/style-guide/images/image-annotations.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Best practices for using image annotations
+contentType: page
+template: basicDoc
+topics:
+  - Basic style guide
+  - Writing guidelines
+---
+
+import styleguideExampleCalloutBox from 'images/style-guide_screenshot-full_example_callout_box.png'
+
+import apisanddataNerdGraphDashboardEntity from 'images/apis-and-data_screenshot-full_NerdGraph-dashboard-entity.png'
+
+import styleguideExampleArrow from 'images/style-guide_screenshot-full_example_arrow.png'
+
+Image annotations are text and shapes in screen captures or diagrams that help to reference information and focus attention on the most important aspects of the screenshot. We recommend that you use them cautiously because sometimes they can be a distraction and are also difficult to keep up to date. Keep in mind that you can use alternatives such as [image captions](/docs/style-guide/images/screenshots-images/#captions) or instructions right in the text.
+
+If you need to use image annotations:
+
+* Be concise.
+* Make sure annotations are easy to distinguish from UI text. For example, you may need to offset your boxes so they don't look like UI elements.
+* Annotations aren't translated, so use them sparingly.
+* Don't use an annotation at the top that acts like a title. Use a caption.
+* Use our [New Relic SnagIt theme](https://github.com/newrelic/newrelic-doc-tools) (link only available to New Relic employees) to keep your annotations consistent.
+
+## Install the New Relic SnagIt theme [#install-theme]
+
+  Follow these steps:
+    1. Ensure you're running the latest version of SnagIt.
+    2. In GitHub Desktop, clone the [repo](https://github.com/newrelic/newrelic-doc-tools) locally.
+    3. Open SnagIt and select any tool to open the right nav.
+    4. Under the **Quick Styles** pane in the right nav, select the **Theme** gear icon, then **Import**.
+
+## Callout boxes [#callout-boxes]
+
+Use callouts to highlight an element within a screenshot. If possible, don't add text to screenshots, that's what our docs are for. Callout boxes should not obstruct any of the elements they're calling out. For example, a callout box around a UI button shouldn't cover any part of the button. If possible, insert a two-pixel clearance between the UI element and the callout box.
+
+This is the format for adding callout boxes:
+  * Shape: square or rectangle, transparent fill, no border-radius.
+  * Border color: #00CE7C
+  * Line width: 5px
```

Comment:
@nbaenam I just realized that SnagIt uses "pt" instead of "px." They are slightly different, so will that affect the value you want? in this case, the value should be "5pt"